localturk.ts

```diff
@@ -159,8 +158,9 @@ async function getNextTask(): Promise<TaskStats> {
 }
 
 const app = express();
-app.use(bodyParser.urlencoded({extended: false}));
 app.use(errorhandler());
+app.use(express.json({limit: "50mb"}));
+app.use(express.urlencoded({limit: "50mb", extended: true, parameterLimit:50000}));
```

Comment:
Is there a reason this needs to change from `extended: false` to `extended: true`? Is it the same `extended`?

Reply:
I'll be honest: I'm not sure, I just pulled it from the answer (it seemed to be consistent across a few answers). [This post](https://stackoverflow.com/questions/55558402/what-is-the-meaning-of-bodyparser-urlencoded-extended-true-and-bodypar) seems to have some more information. Based on that, I think it may be fine to set to `false`?